.github/workflows/generate_app_id.yml

```diff
@@ -61,6 +61,20 @@ jobs:
                         ```
                       </details>
 
+            - name: Retrieve pull request head branch
+              uses: xt0rted/pull-request-comment-branch@v2
+              id: comment-branch
+
+            - name: Notify translations team
+              if: ${{ success() && github.ref_name == 'deriv_app_translations_2' }}
+              run: |
+                content="Paimon has retrieved the generated test link for the updated translations:\n"
+                content+=":paperclip: *Pull Request*: ${{ steps.generate_app_id.outputs.pr_url }}\n"
+                content+=":link: *Test Link*: ${{ steps.vercel_preview_url.outputs.vercel_preview_url }}\n"
+                content+=":id: *App ID*: ${{ steps.generate_app_id.outputs.app_id }}"
+
+                curl -X POST -H 'Content-type: application/json' --data "{\"text\": \"$content\" }" ${{ secrets.SLACK_TRANSLATIONS_TEAM_WEBHOOK }}
```

Comment:
The message to retrieve the test link can only be generated under the `issue_comment` `edited` event, the `repository_dispatch` event does not contain the context for retrieving the test links

Reply:
<img width="716" alt="Screenshot 2023-07-28 at 1 47 45 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/837ec86e-daa0-4b1c-8a36-fa59ae9c01a4">